mlflow/ml-package-versions.yml

```diff
@@ -42,6 +42,17 @@ pytorch:
     run: |
       pytest tests/pytorch/test_pytorch_model_export.py tests/pytorch/test_pytorch_metric_value_conversion_utils.py
 
+torch:
+  package_info:
+    pip_release: "torch"
+    install_dev: |
+      export PACKAGE_NAME=pytorch
+      pip install git+https://github.com/PytorchLightning/pytorch-lightning.git
+  autologging:
+    minimum: "1.4"
+    maximum: "1.14"
```

Comment:
```suggestion
    minimum: "1.14"
    maximum: "1.4"
```

Reply:
4 is less than 14, or am I missing something?

Reply:
Sorry, you're right :)